### DISCUSSION TITLE
R/D

### DISCUSSION THREAD

**Discussion by Kosnov** [kachurovskiy/nanoels#201] [General]
Hello
It is possible to enter data not for radius, but for diameter
Sincerely

**Comment by kachurovskiy** [comment]
Hi @Kosnov, sorry but the radius logic is pretty deeply ingrained in all the modes. There's one place where you can enter diameter though - when zeroing X axis.

https://github.com/kachurovskiy/nanoels/blob/main/h4/README.md#zeroing-the-axes

**Reply by Kosnov** [reply]
Благодарю
